Disher/Models/Recipe.h

```diff
@@ -5,11 +5,17 @@
 //  Created by Bienn Viquiera on 7/7/22.
 //
 
-#import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
+
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface Recipe : NSObject
+@interface Recipe : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSString *dishName;
+@property (nonatomic, strong) NSString *imageURL;
+@property (nonatomic, strong) NSString *source;
+@property (nonatomic) int *recipeID;
```

Comment:
are you aiming to hold a pointer to int (int *) or an integer value (int)?

Reply:
an int value, will fix